makefile

```diff
@@ -3,22 +3,31 @@ SHELL := /bin/bash
 all: crud metrics tracer
 
 crud:
-	export GO111MODULE=on
 	cd "$$GOPATH/src/github.com/ardanlabs/service"
-	docker build -t crud-amd64 -f dockerfile.crud .
-	docker system prune -f
```

Comment:
I want to run prune after the build automatically. It helps to keep my system clean.

Reply:
The problem with `system` pruning after every build is that you destroy the build cache. What artifacts are you concerned about cleaning up? (there are different prune scopes beside system that might give a better result for what you want).